tests/test_fyle/test_tasks.py

```
@@ -1248,3 +1251,243 @@ def test_cleanup_scheduled_task_not_found(
+
+    expense = add_expense_report_data['expenses'][0]
+    report_id = expense.report_id
+
+    mock_delete = mocker.patch('apps.fyle.tasks.delete_accounting_export_and_related_data')
+
+    _delete_accounting_exports_for_report(report_id, workspace)
+
+    assert mock_delete.called
+
+
+def test_delete_accounting_exports_for_report_no_expenses(db, mocker):
```

Ruff (0.14.0)

Unused function argument: `db`

(ARG001)